appcli/crypto/crypto.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+# # -*- coding: utf-8 -*-
+
+"""
+Handles encryption/decryption of data.
+________________________________________________________________________________
+
+Created by brightSPARK Labs
+www.brightsparklabs.com
+"""
+
+# standard libraries
+from os import linesep
+import re
+from pathlib import Path
+
+# vendor libraries
+from Crypto.Random import get_random_bytes
+
+# local libraries
+from appcli.crypto.cipher import Cipher
+
+
+def create_key() -> bytes:
+    return get_random_bytes(32)
+
+
+def create_and_save_key(key_file: Path):
+    key_file.write_bytes(create_key())
```

Comment:
I think this needs to check if the file exists first, as we don't want to write to a non-empty file
We may also want to restrict the permissions on the file after creating it

Reply:
Check if file exists. Don't worry about permissions at this point as we don't understand enough about the target system.